### PR TITLE
PLT-2513 Allow removing of license without turning off enterprise features

### PR DESCRIPTION
**Pull request by jwilander**
I also removed some localization from the Edition and License page since that page should not be localized.

### CODE REVIEW

File: webapp/components/admin_console/license_settings.jsx [hunk 1]
```diff
@@ -145,10 +144,7 @@ class LicenseSettings extends React.Component {
                     <br/>
                     <br/>
                     <p className='help-text'>
-                        <FormattedHTMLMessage
-                            id='admin.licence.keyMigration'
-                            defaultMessage='If you’re migrating servers you may need to remove your license key from this server in order to install it on a new server. To start, <a href="http://mattermost.com" target="_blank">disable all Enterprise Edition features on this server</a>. This will enable the ability to remove the license key and downgrade this server from Enterprise Edition to Team Edition.'
-                        />
+                        {'If you migrate servers you may need to remove your license key to install it elsewhere. You can remove the key here, which will revert functionality to that of Team Edition.'}
```
Comment (crspeller) [inline webapp/components/admin_console/license_settings.jsx]: @jwilander This is help text, not anything to do with licencing.

Comment (crspeller) [inline webapp/components/admin_console/license_settings.jsx]: Going to link this in PM room to confirm.

Comment (jwilander) [inline webapp/components/admin_console/license_settings.jsx]: The original spec asks for all text on Edition and License page to be unlocalized.

Comment (crspeller) [inline webapp/components/admin_console/license_settings.jsx]: PMs want all English.

**Comment by crspeller** [comment]
+1

**Pull request merged**